rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+
+    # associations
+    book.rentals << self
+    person.rental << self
```

Comment:
- [ ] Based on the requested change above, ensure that you have the correct name. Either `person.rental` or `person.rentals`